spec/lavinmqctl_spec.cr

```diff
@@ -0,0 +1,384 @@
+require "./spec_helper"
+require "../src/lavinmqctl/cli"
+
+# Monkey-patch LavinMQCtl to redirect puts/print to captured IO
+class LavinMQCtl
+  def puts(*args, **options)
+    @io.puts(*args, **options)
+  end
+
+  def print(*args)
+    @io.print(*args)
+  end
+end
+
```

Comment:
looks like we're doing `@io.puts` in LavinMQCtl everywhere already? or could this go into the actual LavinMQCtl? 
```suggestion
```